client/src/components/community/ChannelButton.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="channel-button" :class="selected && 'active'">
+    <div>
+      <i data-feather="hash" class="hashtag"></i>
+      <span>{{name}}</span>
+    </div>
+    <div class="actions" v-if="true">
+      <i data-feather="user-plus" class="add"></i>
```

Comment:
```suggestion
      <svg class="add" viewBox="0 0 24 24" width="24" height="24" stroke="currentColor" stroke-width="2" fill="none" stroke-linecap="round" stroke-linejoin="round"><path d="M16 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2"></path><circle cx="8.5" cy="7" r="4"></circle><line x1="20" y1="8" x2="20" y2="14"></line><line x1="23" y1="11" x2="17" y2="11"></line></svg>
```

Reply:
I was wondering about lots of things while doing the layout and kept my fingers out of many things intentionally. Otherwise, CJ won't have anything to show on stream 😉

My intentions with my PR's is create a starting point where CJ can pick where I left off and create content. Well, apart from the Docker one, that's a real fix.

Thanks.

Reply:
This is fine for now. Eventually we will remove feathers icons and use the icons shidotmoe created.